client-lite/src/exe/docs.cpp

```diff
@@ -90,7 +104,11 @@ HRESULT Run() try
 
     DoLogInfo("Started, %s", msdoutil::ComponentVersion().c_str());
 
-    ProcessController procController;
+    std::function<void()> fnRefreshConfigs = [&downloadManager]()
```

Comment:
>    std::function<void()> [](start = 0, length = 25)

Can use auto here but consider writing the lambda inline on line 111. It will also satisfy the && parameter type.

Reply:
Right, able to write inline due to parameter type being an rvalue reference

---
In reply to: [595575443](https://github.com/microsoft/do-client/pull/11#discussion_r595575443) [](ancestors = 595575443)